### PR TITLE
Separating out apps data into rows/columns + adding unit def.

### PR DESCRIPTION
**Pull request by juliaelman**
Refs #306.

This PR completes the list of items defined in these small design fixes.

screen shot 2016-06-08 at 4 08 29 pm

### CODE REVIEW

File: static_src/components/app_list.jsx [hunk 1]
```diff
@@ -69,11 +69,37 @@ export default class AppList extends React.Component {
...
+                <td label="Memory">{ app.memory } MB</td>
+                <td label="Instances">{ app.instances }</td>
+                <td label="State">{ app.state }</td>
+                <td label="Disk quota">{ app.disk_quota } MB</td>
+              </tr>
+            ])
+          })}
+          </tbody>
+        </table>
+      );
```
Comment (jeremiak) [inline static_src/components/app_list.jsx]: Nice, I find this to be much more readable than using `Reactable`.

**Comment by jeremiak** [comment]
This looks great, nice work.

Minor quibble: On the app page (`/#/orgs/:guid/spaces/:guid/apps/:guid`) there isn't any space between the number and "MB" so we should be consistent. I think this would also prevent the text from breaking when the memory quantity is four digits. I think being consistent is more important than preventing line breaks so I think either approach is fine

Screenshot of app page:
Screenshot of app page

Screenshot of line break in app list table:
Screenshot of line break in app list table

File: static_src/components/app_list.jsx (hunk 1 above)
Comment (msecret) [inline static_src/components/app_list.jsx]: Yea we gotta get rid of Reactable at some point, I think it sucks.

**Comment by msecret** [comment]
Speaking of @jeremiak 's comment, I like the way on the app list page more, having a space between the number and MD.

**Comment by jeremiak** [comment]
This looks great. I went ahead and formatted the app page to match the app list page. I'll merge when this is green

**Pull request merged**